server-phoenix/lib/helios_web/schema.ex

```diff
@@ -7,7 +7,12 @@ defmodule HeliosWeb.Schema do
   end
 
   def get_string_key(%{source: source} = res, key) do
-    %{res | state: :resolved, value: Map.get(source, key)}
+    cond do
+      is_map(source) ->
```

Comment:
@linalim and I discovered this can follow a method declaration like so:
```
  def get_string_key(%{source: source} = res, key) when is_map(source) do
    ....
  end
```